nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Kindly add some kind of text to let the user know which class does not have the method.

  ```
  raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  ```